BedrockCore.cpp

```diff
@@ -171,7 +171,6 @@ BedrockCore::RESULT BedrockCore::peekCommand(unique_ptr<BedrockCommand>& command
         response["commitCount"] = to_string(_db.getCommitCount());
 
         // Success. If a command has set "content", encode it in the response.
-        SINFO("Responding '" << response.methodLine << "' to read-only '" << request.methodLine << "'.");
```

Comment:
This is kind of useful, PHP already logs it but we would not have this for all-auth processes though (like billing and such)...

Reply:
We also get a timing log when each command completes. This is only differently useful if we're particularly looking at whether or not we sent a response.

Reply:
But that does not log the response code... though maybe it does not matter as there should always be an exception if we don't return 200?
Anyway, I already approved 😄